src/convert.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { convertSelicEntry } from "./convert";
+import { format, compareAsc } from "date-fns";
+
+describe("convert", () => {
+  test("Successful convert date and value", () => {
+    expect(convertSelicEntry("08/03/2024", "0.041957")).toEqual({
+      date: new Date(format("2024-03-08", "yyyy-MM-dd")),
```

Comment:
Why not use [parse](https://date-fns.org/v3.6.0/docs/parse)?

```suggestion
      date: parse("2024-03-08", "yyyy-MM-dd"),
```